up-linux-streamer/examples/mE_client.rs

```diff
@@ -0,0 +1,108 @@
+use async_trait::async_trait;
```

Comment:
I'm not sure whether the examples should be put under the folder. If I understand correctly, these examples can be used by both up-linux-streamer and up-linux-streamer-plugin, right?

Reply:
Anyway, the folder reorganization can be done in the future.

Reply:
You are correct. It may be better to move the examples out. I'll think on it.

Reply:
I will go ahead and move them to their own crate.

Reply:
Moved them